### ISSUE TITLE
Connectivity issues with local Prism Agents on *nix OS

### ISSUE THREAD

**Issue by lohanspies** [lohanspies/prism_v2_playground#4]
It seems `docker.host.internal` doesn't work on *nix-based operating systems. Had issues with Ubuntu and NixOS.
We need to investigate and find a solution to the problem.

Environment config:
```
Inviter URL:http://host.docker.internal:8080/prism-agent
Invitee URL:http://host.docker.internal:8090/prism-agent
```

Error:
```
ConnectError: [Errno -2] Name or service not known
```

**Comment by lohanspies** [comment]
It seems to work when using `172.17.0.1` instead of `docker.host.internal`.
Still determining how it will influence DIDComm communication. I need to do further tests.

**Comment by lohanspies** [comment]
Added https://github.com/lohanspies/prism_v2_playground/blob/master/work/BetaProgram/variables-linux.env to be used when the host OS is linux based.

**Comment by lohanspies** [comment]
For some background information:
https://stackoverflow.com/questions/74523100/how-can-i-find-the-host-os-from-a-docker-container

```
I don't think there's a documented way, but there is an OS-specific way that DNS lookups for host.docker.internal and docker.for.mac.localhost are handled. Specifically, on Docker Desktop for Mac, both of those exist; on Docker Desktop for Windows, only the first one exists, and on Docker Engine for Linux, neither exist. – 
[Nick ODell](https://stackoverflow.com/users/530160/nick-odell)
 [Nov 21, 2022 at 18:22](https://stackoverflow.com/questions/74523100/how-can-i-find-the-host-os-from-a-docker-container#comment131551327_74523100)
 ```

**Comment by lohanspies** [comment]
Provided an initial fix to this problem with two commits [3ca8556734497f7fc4a70ad2208db05f15b05a9b](https://github.com/lohanspies/prism_v2_playground/commit/3ca8556734497f7fc4a70ad2208db05f15b05a9b) and [f4d171c3e8d1c2fe07f9cf1db669e5560497ba4d](https://github.com/lohanspies/prism_v2_playground/commit/f4d171c3e8d1c2fe07f9cf1db669e5560497ba4d)
